array.h

```diff
@@ -982,6 +982,10 @@ NDARRAY_HOST_DEVICE shape<Dims...> make_shape_from_tuple(const std::tuple<Dims..
 template <size_t Rank>
 using index_of_rank = internal::tuple_of_n<index_t, Rank>;
 
+// ELEPHANT
```

Comment:
Don't submit this yet. Introduced to try constructing a `shape` from a `bounds`.